### ISSUE TITLE
NVDA 2010.1 interaction with web based e-mail service by GMX.com.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#696]
**Reported by H.D. on 2010-06-11 18:40**
When accessing gmx.com e-mail via web site (be it using Firefox or IE of latest release), the NVDA 2010.1 does not recognize anything on the page after user is logged-in; that is, the interface of it cannot be access or read by NVDA. Reportedly, the gmx.com web based e-mail uses "web two" specifications; also, it does not place its e-mail navigations in hyper links (i.e. unlike we are used to in GMail). The entire screen appears black -- or at least, that is what the NVDA says. I've tried it with the latest NVDA snapshots, the result is the same. Could it be because of "web 2" implementation by GMX.com (which I'm not sure if NVDA support or not)?

**Comment by leonardder** [comment]
This is probably not a bug in NVDA. Furthermore, the last reply dates from seven years ago. Won't fix?
